src/Symfony/Bridge/Doctrine/Validator/Constraints/UniqueEntityValidator.php

```diff
@@ -71,6 +71,7 @@ public function validate(mixed $entity, Constraint $constraint)
         if ($constraint->em) {
             $em = $this->registry->getManager($constraint->em);
 
+            // doctrine/persistence < 4 compat
```

Comment:
this comment is actually wrong. Older versions of `doctrine/persistence` are also throwing an exception in `getManager`